gov/politeia/types/types.go

```diff
@@ -240,14 +240,12 @@ type ProposalMetadata struct {
 // prepare figures for display. Many of these manipulations require a tip height
 // and a target block time for the network, so those must be provided as
 // arguments.
-func (pinfo *ProposalInfo) Metatdata(tip, targetBlockTime int64) *ProposalMetadata {
+func (pinfo *ProposalInfo) Metadata(tip, targetBlockTime int64) *ProposalMetadata {
 	meta := new(ProposalMetadata)
 	desc := pinfo.VoteStatus.ShortDesc()
 	switch desc {
 	case "Authorized":
-		blocksTil := windowSize - tip%windowSize
-		meta.StartHeight = tip + blocksTil
-		meta.SecondsTil = blocksTil * targetBlockTime
+		meta.SecondsTil = int64(windowSize) * targetBlockTime
```

Comment:
Sorry @dmigwi. I was probably not being very clear here. We have no reason to set this field, and doing so is a little misleading. Seconds until what? This value, as you've set it, is a constant. It's just the size of the window converted to time, and is purposefully set to be 7 days. It is not the time til voting starts or ends or until any other time relevant to the metadata for an authorized proposal.

For the metadata, we expect some fields to be unset during certain stages of the proposal process. You can just drop the "Authorized" case altogether here.